src/strategies/xdai-stake-lp/index.ts

```diff
@@ -0,0 +1,149 @@
+import { formatUnits } from '@ethersproject/units';
+import { getAddress } from '@ethersproject/address';
+import { multicall, subgraphRequest } from '../../utils';
+import examplesFile from './examples.json';
+
+export const author = 'maxaleks';
+export const version = '0.1.0';
+export const examples = examplesFile;
```

Comment:
Examples JSON imported automatically now (with new changes) :) you can remove this
```suggestion
```